extension/src/state_aggregate.rs

```diff
@@ -185,6 +189,18 @@ pub fn duration_in(state: String, aggregate: Option<StateAgg>) -> i64 {
         .unwrap_or(0)
 }
 
+#[pg_extern(immutable, parallel_safe, schema = "toolkit_experimental")]
+pub fn values(
```

Comment:
Recommend changing the name to `display` (or if you come up with anything better, just not values).

Reply:
I also wonder whether `display` is good choice either since it might suggest for display only. I don't have a good suggestion either. The best that I came up is `unpack`.

Reply:
`compute`? `capture`? `obtain`? `manifest`? `embody`? `occur`? `appear`? `emerge`? `unfold`? `objectify`?

Agree with @jerryxwu that `display` connotes a particular context that might suggest function is more limited that it actually is.